.github/workflows/_build-packages.yml

```diff
@@ -63,8 +64,9 @@ jobs:
           mkdir pypi/${{ matrix.pkg-name }}
           cp dist/* pypi/${{ matrix.pkg-name }}/
 
-      - uses: actions/upload-artifact@v3
+      - uses: actions/upload-artifact@v4
         with:
           name: ${{ inputs.artifact-name }}
           path: pypi
           include-hidden-files: true
+          overwrite: true
```

Comment:
Artifacts are uploaded in the init job (line 28) as a placeholder, therefore we need to overwrite them. See https://github.com/Lightning-AI/pytorch-lightning/actions/runs/12359005253/job/34490799219#step:8:22